_posts/2022-04-12-identity.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "How Flotta project proofs identity on the far edge"
+date:   2022-04-12 12:00:00 +0100
+categories: flotta
+author: Eloy Coto <eloy.coto@acalustra.com>
+tags:
+  - authentication
+  - flotta
+
+summary-1: How Flotta project proofs identity on the far edge.
+---
+
+When speaking about the edge one of the most problematic things is the security
+point of view. At the edge, you cannot assume there is security at all, so we
+assume that all your devices are compromised by default and we adopt a [Zero
+trust](https://cloud.google.com/beyondcorp) overview.
+
+Edge Authentication is getting traction, and at the moment, there are multiple
+projects out there that it’s taking care of it, for us, the most promised one,
+is the protocol-driven by the [FIDO alliance](https://fidoalliance.org/),
+regarding Device Onboarding, also known as
+[FDO](https://fidoalliance.org/specs/FDO/FIDO-Device-Onboard-RD-v1.0-20201202.html).
```

Comment:
Do you think it makes sense to write few words about what FDO is and how it will be used in flotta in the future?